CDP4ShellDialogs/ViewModels/ModelIterationDomainSwitchDialogViewModel.cs

```diff
@@ -108,6 +108,7 @@ private async Task ExecuteDomainSwitch()
                 if (openIteration != null)
                 {
                     session.SwitchDomain(modelrow.IterationIid, modelrow.SelectedDomain);
+                    modelrow.ActiveParticipant.SelectedDomain = modelrow.SelectedDomain;
```

Comment:
Why do you need this? The switching should already have happened and the dialog is presumably closing.